packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -36,32 +36,25 @@ export function getMomentTemplateBanner(
         const { isReminderActive, onReminderCtaClick, mobileReminderRef } = useReminder(
             reminderTracking,
         );
-
         const isTabletOrAbove = useMediaQuery(from.tablet);
-
-        const mainOrMobileContent = isTabletOrAbove ? 'mainContent' : 'mobileContent';
+        const mainOrMobileContent = isTabletOrAbove ? content.mainContent : content.mobileContent;
 
         return (
             <div css={styles.outerContainer(templateSettings.containerSettings.backgroundColour)}>
-                <Container
-                    cssOverrides={styles.mobileStickyHeaderContainer(
-                        templateSettings.containerSettings.backgroundColour,
-                        content.mobileContent.secondaryCta?.type ===
-                            SecondaryCtaType.ContributionsReminder,
-                        templateSettings.containerSettings.paddingTop,
-                    )}
-                >
+                <Container cssOverrides={styles.containerOverrides}>
```

Comment:
The "mobile sticky header" was one other reason for the duplicate code. A mobile heading and image were placed in a container together in order to use `position: sticky` for them as a unit.

In this PR I have instead opted to keep both components separate and include comments on css styling interdependencies.